harness/features/variable.local.test.ts

```diff
@@ -136,39 +137,46 @@ describe('Variable Tests - Local', () => {
                         expectEventBody(eventResult.body, key, 'aggVariableEvaluated')
                     })
 
-                    it('should return default value if default type doesn\'t match variable type',  async () => {
-                        const eventResult = interceptEvents(name, eventsUrl)
-
-                        const wrongTypeDefault = type === 'number' ? '1' : 1
-                        const variableResponse =
-                            await testClient.callVariable(
-                                { user_id: 'user1', customData: { 'should-bucket': true } },
-                                key,
-                                wrongTypeDefault
-                            )
-                        const variable = await variableResponse.json()
-
-                        // Expect that the test returns a defaulted variable
-                        expectDefaultValue(key, variable, wrongTypeDefault,
-                            wrongTypeDefault === '1' ? VariableType.string : VariableType.number)
-
-                        if (!hasCapability(name, Capabilities.events)) {
-                            return
-                        }
+                    // Skip test for OpenFeature NodeJS SDK as it throws the error in the OF Provider
+                    // and doesn't log events which doesn't seem like something we can easily disable checking for.
+                    if (name !== 'OF-NodeJS') {
```

Comment:
tried this test by disabling `interceptEvents()` and exiting at `hasCapability` below, but got error where it was expecting a `/events/batch` call still.

Reply:
I think you might need to remove `Capabilities.events` from the corresponding record in capabilities.ts?